packages/shared/src/hooks/squads/usePostToSquad.ts

```diff
@@ -149,10 +171,33 @@ export const usePostToSquad = ({
     [preview, displayToast, onSubmitLink, onPost, isPosting],
   );
 
+  const onUpdatePost = useCallback(
+    (
+      e: BaseSyntheticEvent,
+      postId: Post['id'],
+      sourceId: Squad['id'],
+      commentary: string,
```

Comment:
This is basically double-defined in the interface and here, can't we make it 1 type to share across both instances?

Reply:
good point, cleaned it up 🙏